scripts/Makefile.depends

```diff
@@ -0,0 +1 @@
+# autogenerated
```

Comment:
I don't know why this line shows as added here when it's present in develop too, but seems OK